ovos_audio/playback.py

```
@@ -27,6 +32,13 @@
         self._now_playing = None
         self._started = Event()
         self.tts_transform = TTSTransformersService(self.bus)
+        self.g2p = None
+        if Configuration().get("g2p", {}).get("module"):
+            LOG.debug("Loading Grapheme2Phoneme plugin for mouth movements")
+            try:
+                self.g2p: Optional[Grapheme2PhonemePlugin] = OVOSG2PFactory.create()
+            except:  # not mission critical
+                pass
```

Ruff

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)